wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -218,6 +244,11 @@ public TypeName typeName(ProtoType protoType) {
     return candidate;
   }
 
+  public TypeName protoFieldsTypeName(ProtoType protoType) {
```

Comment:
Add a short doc explaining what this returns and what it means if the result is null?